test/integration/default/serverspec/server_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'serverspec'
+
+set :backend, :exec
+
+describe command('curl http://127.0.0.1:4000/directory') do
+  its(:stdout) { should match(%r{"new-cert": "http://127.0.0.1:4000/acme/new-cert"}) }
```

Comment:
We can change `127.0.0.1` to `boulder` to ensure the hostfile alias defined in the recipe works properly